django/db/models/expressions.py

```diff
@@ -482,8 +482,11 @@ class Func(Expression):
     function = None
     template = '%(function)s(%(expressions)s)'
     arg_joiner = ', '
+    arity = None
```

Comment:
If we continue with the word arity, let's provide a quick comment definition here for those unfamiliar with the term.

"number of expressions the function requires" or similar.